feat: stkGHO Incentives Q4 (#506)
bgd-labs/aave-proposals-v3: src/20241029_AaveV3Ethereum_StkGHOIncentivesQ4/StkGHOIncentivesQ4_20241029.md

---
title: "stkGHO Incentives"
author: "karpatkey_TokenLogic_ACI"
discussions: "[https://governance.aave.com/t/arfc-amend-safety-module-emissions/16640](https://governance.aave.com/t/arfc-safety-module-stkgho-re-enable-rewards/19626)"
snapshot: "https://snapshot.org/#/aave.eth/proposal/0x0f73500d0f65c72482d352080ea9aa0aa92264eb059b8f646cf6f0e86556bc3d"
---

## Simple Summary

Re-enable stkGHO incentives emissions for 180 days after previous period ended.

> On the previous proposal to enable stkGHO emissions (Proposal 91 on the forum), an error in accounting made it so ~2,750 in emissions were not distributed during the last period. This current proposal includes that approval to give out those emissions in addition to the 100 per day.

Reviewer: ```suggestion
On the previous proposal to enable stkGHO emissions (Proposal 91 on the forum), an error in accounting made it so ~2,750 in emissions were not claimable during the last period. This current proposal includes that approval so that those emissions can be claimed, in addition to the 100 per day.
```
